doc/stdenv/platform-support-tiers.chapter.md

```diff
@@ -0,0 +1,87 @@
+# Platform Support Tiers {#platform-support-tiers}
+
+In the context of Nixpkgs, a *platform* can mean a CPU architecture, a
+[cross-compilation](#chap-cross) target or a C library option.
+
+Nixpkgs defines 7 platform *support tiers*, meaning levels of support for each platform.
+
+**TODO**: change build successfully to a definition of "work".
+
+## Tier 1 {#platform-support-tier-1}
+
+These platforms receive the highest level of support, meaning that every
+platform-compatible package in Nixpkgs must successfully build in the Hydra and OfBorg CIs.
+**TODO**: this seems to not be entirely true for Hydra, but true for OfBorg.
+
+- `x86_64-linux`, `gcc` + `glibc`
+
+## Tier 2 {#platform-support-tier-2}
+
+Many of the packages are required to build successfully on these platforms in CI, the rest are supported on a best-effort basis by dedicated platform maintainers.
+**TODO**: link teams
```

Comment:
Problem is that the `@NixOS/aarch64-maintainers` mandated by the RFC was never created